src/views/FrontPage.vue

```diff
@@ -152,43 +152,43 @@ export default Vue.extend({
   },
 
   computed: {
-    adjmatrixLink(): string {
-      return `${this.apps.filter((d) => d.name === 'Adjacency Matrix')[0].url}`;
+    multiMatrixURL(): string {
+      return `${this.apps.filter((d) => d.name === 'MultiMatrix')[0].url}`;
     },
 
-    nodelinkLink(): string {
-      return `${this.apps.filter((d) => d.name === 'Node-Link Diagram')[0].url}`;
+    multiLinkURL(): string {
+      return `${this.apps.filter((d) => d.name === 'MultiLink')[0].url}`;
     },
 
     samples(): Array<{title: string; image: NodeRequire; text: string; href: string }> {
       return [
         {
-          title: 'Paul Revere - Node Link Diagram',
+          title: 'Paul Revere - MultiLink',
           // eslint-disable-next-line global-require
           image: require('../assets/placard/boston.jpg'),
-          text: 'Explore the Paul Revere dataset using an interactive and beautiful node-link diagram. Discover the figures coordinating a pivotal event in history!',
-          href: `${this.nodelinkLink}/?workspace=boston&graph=boston`,
+          text: 'Explore the Paul Revere dataset using an interactive and beautiful MultiLink diagram. Discover the figures coordinating a pivotal event in history!',
```

Comment:
This instance should remain as it was (`MultiLink diagram` doesn't really mean anything, and in this description it's ok to use established vis vocabulary to say what the user will be seeing).

```suggestion
          text: 'Explore the Paul Revere dataset using an interactive and beautiful node-link diagram. Discover the figures coordinating a pivotal event in history!',
```

Reply:
I thought the whole point of the MultiLink app was that it's not just a node-link diagram. I was trying to be more precise with the wording and not set people up to expect a standard node-link diagram

Reply:
That is true but calling it a "MultiLink diagram" doesn't really convey any information. It is in fact a node-link diagram, plus some extras. For an intro text blurb that is meant to get you to click on something, "node-link diagram" is the correct level of description for MultiLink.